openproblems/tasks/cell_cell_communication/__init__.py

```diff
@@ -0,0 +1,12 @@
+from ... import utils
+from . import api
+from . import datasets
+from . import methods
+from . import metrics
+
+_task_name = "Cell-Cell Communication Inference"
+_task_summary = "TODO"
```

Comment:
Please fill in a short (<80 char) summary

Reply:
Done.